app/test_main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from typing import Any
+import pytest
+
+
+import app.main
+
+
+def test_should_access_google_page(monkeypatch: Any) -> None:
+    monkeypatch.setattr(app.main, "has_internet_connection", lambda: True)
+    monkeypatch.setattr(app.main, "valid_google_url", lambda url: True)
+    assert app.main.can_access_google_page("google.com") == "Accessible"
```

Comment:
Is this test almost same as `test_should_not_access_google_page` ? You just can parametrize param `Accessible` or `Not accessible`